syntax = "proto3";

option go_package = "github.com/cilium/proxy/go/cilium/api;cilium";

package cilium;

message NetworkFilter {
  // Path to the proxylib to be opened
  string proxylib = 1;

  // Transparent set of parameters provided for proxylib initialization
  map<string, string> proxylib_params = 2;

  // L7 protocol identifier
  string l7_proto = 3; // Deprecated

  // Name of the policy to be enforced
  string policy_name = 4; // Deprecated

  // Path to the unix domain socket for the cilium access log.
  string access_log_path = 5;
}
